lib/Dialect/TritonGPU/IR/Types.cpp

```diff
@@ -120,11 +120,11 @@ LogicalResult MemDescType::verify(function_ref<InFlightDiagnostic()> emitError,
       return emitError() << "rank must be 2 or 3";
     }
     auto bitwidth = elementType.getIntOrFloatBitWidth();
-    if (!enc.getUnpacked() && bitwidth != 16) {
-      return emitError() << "bitwidth must be 16 for packed tensor memory";
+    if (!enc.getUnpacked() && bitwidth > 16) {
+      return emitError() << "bitwidth must be <= 16 for packed tensor memory";
```

Comment:
This was something I wanted to make more strict regardless. Any chance we could change the use case to leave it as it is?

Reply:
I'm looking at the use case and it's passing fp8 as the lhs in a tcgen05_mma so there is no other way to do it. We should just fix load and store if they really are broken.